### DISCUSSION TITLE
Is McUtils a serious extension?

### DISCUSSION THREAD

**Discussion by sfederici** [ScratchAddons/ScratchAddons#6870] [Discussion]
I can imagine being a fastfood employee, but I can't imagine blocks of the McUtils extension being useful to me in any way even in that case. Is this a serious extension? Or is it just a joke?

**Comment by CubesterYT** [comment]
@sfederici, the McUtils extension is indeed a joke extension. It's basically just a reskinned, funny version of the couplers extension.

**Comment by CubesterYT** [comment]
But personally, this is a question fit asking on the TurboWarp Extension repository, instead of Scratch Addons.

**Reply by sfederici** [reply]
Yes, you are right. Sorry, wrong repository.

**Comment by GarboMuffin** [comment]
It's a real extension up on the website. It's perhaps the only outright "joke extension" on the list. Localizing it was definitely not something that was considered when it was added.

I will be adding a system for adding more context to those names as some of them are not obvious

**Reply by sfederici** [reply]
> I will be adding a system for adding more context to those names as some of them are not obvious

Loading the uncertain extensions and having a look at their blocks is usually enough
